srcs/incs/HttpPostResponse.hpp

```diff
@@ -33,6 +34,7 @@ class HttpPostResponse: public HttpBaseResponse {
 		void	Upload_(File file);
 		void	HandleCGI_(File file);
 		void	HandleUpload_(File file);
```

Comment:
Is it necessary in all of these to pass the File by copy?

Reply:
Not really, I've changed it.